libqtile/confreader.py

```diff
@@ -99,6 +99,20 @@ def update(self, *, fake_screens=None, **settings):
                 value = getattr(self, key, default[key])
             setattr(self, key, value)
 
+    def _reload_config_submodules(self, path):
+        """Reloads python files from same folder as config file."""
+        folder = path.parent
+        for name, module in sys.modules.copy().items():
```

Comment:
Looks like we don't need `name`:
```suggestion
        for module in sys.modules.copy().values():
```